src/node/desktop/src/main/desktop-utils.ts

```diff
@@ -0,0 +1,1026 @@
+/*
+ * desktop-utils.ts
+ *
+ * Copyright (C) 2021 by RStudio, PBC
+ *
+ * Unless you have received this program directly from RStudio pursuant
+ * to the terms of a commercial license agreement with RStudio, then
+ * this program is licensed to you under the terms of version 3 of the
+ * GNU Affero General Public License. This program is distributed WITHOUT
+ * ANY EXPRESS OR IMPLIED WARRANTY, INCLUDING THOSE OF NON-INFRINGEMENT,
+ * MERCHANTABILITY OR FITNESS FOR A PARTICULAR PURPOSE. Please refer to the
+ * AGPL (http://www.gnu.org/licenses/agpl-3.0.txt) for more details.
+ *
+ */
+
+
+import { Xdg } from '../core/xdg';
+import { getenv } from '../core/environment';
+
+export namespace Desktop {
```

Comment:
What is the motivation for using a namespace here (rather than just having these functions exported as part of the module, or defining an explicit export object or similar)?

Reply:
Just playing with different approaches, honestly. Given you can just import everything from a file and give it a namespace at that point, this is probably not a worthwhile pattern. Just removed it.